_maps/map_files/IceBoxStation/IcemoonUnderground_Above.dmm

```diff
@@ -331,9 +331,15 @@
 /turf/open/floor/plasteel,
 /area/quartermaster/warehouse)
 "ba" = (
-/obj/structure/closet/crate/freezer,
 /obj/machinery/power/apc/auto_name/east,
 /obj/structure/cable,
+/obj/structure/closet/crate/secure/freezer{
+	desc = "An insulated crate with a lock on it, used to secure pizza.";
+	name = "secure pizza crate";
+	req_access = list(28);
+	tamperproof = 1
+	},
```

Comment:
Can you make an object variant instead of var-editing this?
Especially the tamperproof variable.